hpedockerplugin/volume_manager.py

```diff
@@ -1045,6 +1058,10 @@ def get_volume_snap_details(self, volname, snapname, qualified_name):
                 'mount_conflict_delay')
             vol_detail['cpg'] = volinfo.get('cpg')
             vol_detail['snap_cpg'] = volinfo.get('snap_cpg')
+
+            if '3par_vol_name' in volinfo:
```

Comment:
@wdurairaj This inspect will work only with 3.0 volumes. For older volumes, this change will not display backend 3PAR name.
When we import volume, we are not updating etcd record with its 3par backend name.

I suggest when we do inspect on volume, can we retrieve the backend name information associated with volume and snapshot?

Reply:
'3par_vol_name' field update for Older volume(s) is addressed now.